bus-mapping/src/evm/opcodes/mstore.rs

```diff
@@ -134,4 +148,77 @@ mod mstore_tests {
 
         Ok(())
     }
+
+    #[test]
+    fn mstore8_opcode_impl() -> Result<(), Error> {
+        let code = bytecode! {
+            .setup_state()
+            PUSH1(0x12)
```

Comment:
I suggest having a value that takes more than 1 byte in this test, so that we can see the effect of `& 0xff`.  For example `PUSH2(0x1234)`.